apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -172,6 +172,34 @@ public object Clone()
         /// <inheritdoc />
         public override int ExecuteNonQuery() => ExecuteNonQueryAsync(_synchronousCancellationTokenSource.Token).ResultWithUnwrappedExceptions();
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings)
+        {
+            return ExecuteReaderAsync(singleUseReadSettings, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public async Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings, 
+            CancellationToken cancellationToken)
```

Comment:
because "None" is not a constant and I don't want to use default(CancellationToken) and assume that is None (which it IS, but I dont want to rely on it)